definitions/verify.v2.yml

```
@@ -12,18 +12,26 @@ servers:
 security:
   - basicAuth: []
   - bearerAuth: []
-tags:
-  - name: Requests
-    description: Verify Requests
 
 paths:
   /:
     post:
       operationId: newRequest
       summary: Request a verification be sent to a user
-      description: Start verifying that we can reach this user given a set of contact details
-      tags:
-        - Requests
+      description: >-
+        Start verifying that we can reach this user given a set of contact details
+
+        <div class="Vlt-callout Vlt-callout--critical">
+          <div class="Vlt-callout__content">
+            <h4>Note on Email & WhatsApp</h4>
+            If you wish to use your own WhatsApp Business Account, or your own email address when using the `email`
```

The sentence construction seems a little unclear to me. Perhaps something like:

```suggestion
            If you wish to use your own WhatsApp Business Account for the WhatsApp channel, or your own email address when using the `email` channel, when populating the  in the 'from' field, please note the following:
```